### PR TITLE
Added instant in personal vehicle with only globals

### PR DESCRIPTION
**Pull request by MoneyDropLobby**
Have fun ;)

### CODE REVIEW

**Review by Yimura — APPROVED** [review]

**Comment by Yimura** [comment]
@MoneyDropLobby fix compile errors and I'll merge it into master.

**Comment by MoneyDropLobby** [comment]
> 

Just fixed it

File: BigBaseV2/src/util/vehicle.hpp [hunk 1]
```diff
@@ -6,6 +6,7 @@
 #include "pointers.hpp"
 #include "script.hpp"
 #include "teleport.hpp"
+#include <script_global.hpp>
```
Comment (Yimura) [inline BigBaseV2/src/util/vehicle.hpp]: Any reason you used < > instead of " " for the include?

Comment (MoneyDropLobby) [reply BigBaseV2/src/util/vehicle.hpp]: Actually no, I just copied my shit over didn't see but if it triggers you, change it

**Review by Yimura — APPROVED** [review]

**Pull request merged**